tests/testthat/_snaps/map2.md

```diff
@@ -3,13 +3,17 @@
     Code
       map2_int(1, 1, ~"x")
     Condition
-      Error:
-      ! Can't coerce element 1 from a character to a integer
+      Error in `map2_int()`:
+      ! Can't compute index 1.
+      Caused by error:
+      ! Can't coerce from a character to a integer.
```

Comment:
should be "an integer". You can use the `obj_type_friendly()` callable from rlang if that's helpful.